closure/templates/test/BUILD

```diff
@@ -34,6 +34,28 @@ closure_js_template_library(
     srcs = ["greeter.soy"],
 )
 
+closure_js_template_library(
+    name = "localized_soy",
+    srcs = ["localized.soy"],
+    should_generate_soy_msg_defs = True,
+)
+
+file_test(
+    name = "localized_test",
+    size = "small",
+    file = ":localized.soy.js",
+    regexp = "goog.getMsg('Hello",
+)
+
+
+file_test(
+    name = "nonlocalized_test",
+    size = "small",
+    file = ":greeter.soy.js",
+    regexp = 'goog.getMsg("Hello',
```

Comment:
You quotes are inconsistent between two regexes. It doesn't seem right.

Reply:
Good catch, updated. It actually is enough to check for absence of goog.getMsg, without caring about the argument.